fixtures/ayra/ero-micro-beam-fx.json

```
@@ -0,0 +1,229 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "ERO Micro Beam FX",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["M. Peters"],
+    "createDate": "2024-06-21",
+    "lastModifyDate": "2024-06-21"
+  },
+  "links": {
+    "manual": [
+      "https://www.bax-shop.nl/index3.php?option=com_download_files&product_id=514480&download_id=170502"
+    ],
+    "productPage": [
+      "https://www.bax-shop.nl/moving-head/ayra-ero-micro-beam-fx"
+    ],
+    "video": [
+      "https://www.bax-shop.nl/moving-head/ayra-ero-micro-beam-fx#video"
+    ]
+  },
+  "physical": {
+    "dimensions": [200, 140, 240],
+    "weight": 2.6,
+    "power": 35,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "1x 15W RGBW multichip LED en 24x 5050RGB SMD LEDs"
+    },
+    "lens": {
+      "degreesMinMax": [5, 25]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "StrobeSpeed",
+        "speedStart": "fast",
+        "speedEnd": "slow"
+      }
+    },
+    "Pan": {
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "540deg"
+      }
+    },
+    "Tilt": {
+      "capability": {
+        "type": "Tilt",
+        "angleStart": "0deg",
+        "angleEnd": "270deg"
+      }
+    },
+    "Pan/Tilt Speed": {
+      "capability": {
+        "type": "PanTiltSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
```

```suggestion
    "White Beam": {
```

This is called just "White" in the DMX table, but only the beam has a white LED, so I think it's clearer if we label this consistently.